x-pack/plugins/security_solution/public/timelines/components/field_renderers/field_renderers.tsx

```diff
@@ -281,91 +284,50 @@ interface DefaultFieldRendererOverflowProps {
 }
 
 interface MoreContainerProps {
-  attrName?: string;
-  dragDisplayValue?: string;
-  fieldType?: string;
+  fieldName: string;
+  fieldType: string;
+  values: string[];
   idPrefix: string;
-  isAggregatable?: boolean;
   moreMaxHeight: string;
   overflowIndexStart: number;
   render?: (item: string) => React.ReactNode;
-  rowItems: string[];
 }
 
-/** A container (with overflow) for showing "More" items in a popover */
 export const MoreContainer = React.memo<MoreContainerProps>(
-  ({
-    attrName,
-    dragDisplayValue,
-    fieldType,
-    idPrefix,
-    isAggregatable,
-    moreMaxHeight,
-    overflowIndexStart,
-    render,
-    rowItems,
-  }) => {
+  ({ fieldName, fieldType, idPrefix, moreMaxHeight, overflowIndexStart, render, values }) => {
+    const { timelineId } = useContext(TimelineContext);
+
     const moreItemsWithHoverActions = useMemo(
       () =>
-        rowItems.slice(overflowIndexStart).reduce<React.ReactElement[]>((acc, rowItem, index) => {
-          const id = escapeDataProviderId(`${idPrefix}-${attrName}-${rowItem}-${index}`);
-          const dataProvider =
-            typeof rowItem === 'string' && attrName != null
-              ? {
-                  and: [],
-                  enabled: true,
-                  id,
-                  name: rowItem,
-                  excluded: false,
-                  kqlQuery: '',
-                  queryMatch: {
-                    field: attrName,
-                    value: rowItem,
-                    displayValue: dragDisplayValue ?? rowItem,
-                    operator: IS_OPERATOR as QueryOperator,
-                  },
-                }
-              : undefined;
+        values.slice(overflowIndexStart).reduce<React.ReactElement[]>((acc, value, index) => {
+          const id = escapeDataProviderId(`${idPrefix}-${fieldName}-${value}-${index}`);
 
-          if (dataProvider != null) {
+          if (typeof value === 'string' && fieldName != null) {
             acc.push(
               <EuiFlexItem key={`${idPrefix}-${id}`}>
```

Comment:
Found a mistake done by myself here, I think we could remove the idPrefix in here, as id has included idPrefix.

Reply:
Good catch. Done!